docs/index.html

```diff
@@ -46,6 +46,11 @@ <h2 class="title">A 1x Engineer...</h2>
         <li>Supports code in production, even if they did not write it.</li>
         <li>Can feel like an impostor at times.</li>
         <li>Makes sure discussions are always productive and everyone gets to have their say before the team makes a decision. Only disregards opinions by providing arguments.</li>
+        <li>Create community and share your knowledge</li>
+        <li>Never stop learning</li>
+        <li>Always leave your comfort zone</li>
+        <li>Contribute to the community</li>
+        <li>Document your code so that others can understand it when they use it</li>
```

Comment:
I think this one may already be covered in lines 34/35 (reads the docs/updates the docs) but if you feel it's significantly different I'm more than willing to ship it.

```suggestion
        <li>Document your code so that others can understand it when they use it.</li>
```